app/models.py

```diff
@@ -2962,3 +2965,49 @@ def serialize(self) -> SerializedServiceJoinRequest:
             reason=self.reason,
             contacted_service_users=[get_uuid_string_or_none(user.id) for user in self.contacted_service_users],
         )
+
+
+@dataclass
+class SerializedReportRequest:
+    id: str
+    user_id: str
+    service_id: str
+    report_type: str
+    status: str
+    parameter: dict
+    created_at: str
+    updated_at: str
+
+
+class ReportRequest(db.Model):
+    __tablename__ = "report_requests"
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    user_id = db.Column(UUID(as_uuid=True), db.ForeignKey("users.id"), nullable=False)
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey("services.id"), nullable=False)
+    report_type = db.Column(
+        db.Enum(*REPORT_REQUEST_REPORT_TYPES, name="report_request_report_type"),
+        nullable=False,
+    )
+    status = db.Column(
+        db.Enum(*REPORT_REQUEST_STATUS_TYPES, name="report_request_status_type"),
+        nullable=False,
+        default=REPORT_REQUEST_REQUESTED,
+    )
+    parameter = db.Column(JSONB, nullable=False, default={})
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
+    updated_at = db.Column(
+        db.DateTime, nullable=False, default=datetime.datetime.utcnow, onupdate=datetime.datetime.utcnow
```

Comment:
`updated_at` should be nullable, because it won't be set when the row is first created.